tests/clickhouse-test

```diff
@@ -130,7 +130,7 @@ def run_single_test(args, ext, server_logs_level, client_options, case_file, std
         database = 'test_{suffix}'.format(suffix=random_str())
 
         clickhouse_proc_create = Popen(shlex.split(args.client), stdin=PIPE, stdout=PIPE, stderr=PIPE)
```

Comment:
Maybe `Text=True` argument will help to avoid encoding of input and output lines

Reply:
Indeed